examples/render_text/app/main.py

```diff
@@ -20,14 +23,46 @@ async def chat_completion(self, request: Request, response: Response):
             # Get the last message content
             content = request.messages[-1].content or ""
 
+            # The image may be returned either as base64 string or as URL
```

Comment:
@adubovik Could you please add the same information to the readme?

Reply:
It's already documented in the README